### DISCUSSION TITLE
ImageMagick for Android

### DISCUSSION THREAD

**Discussion by Jay123210599** [ImageMagick/ImageMagick#7456] [Help]
### ImageMagick version

7.1.1-27

### Operating system, version and so on

Windows 11

### Description

Is there a way to use ImageMagick for Android mobile devices?

**Comment by exekutive** [comment]
for what?

**Reply by Jay123210599** [reply]
For my previous tasks

**Reply by exekutive** [reply]
such as?

**Reply by Jay123210599** [reply]
All of these.

https://github.com/ImageMagick/ImageMagick/discussions/7101
https://github.com/ImageMagick/ImageMagick/discussions/7135
https://github.com/ImageMagick/ImageMagick/discussions/7158
https://github.com/ImageMagick/ImageMagick/discussions/7159
https://github.com/ImageMagick/ImageMagick/discussions/7175
https://github.com/ImageMagick/ImageMagick/discussions/7177
https://github.com/ImageMagick/ImageMagick/discussions/7303
https://github.com/ImageMagick/ImageMagick/discussions/7321
https://github.com/ImageMagick/ImageMagick/discussions/7361
https://github.com/ImageMagick/ImageMagick/discussions/7363
https://github.com/ImageMagick/ImageMagick/discussions/7364
https://github.com/ImageMagick/ImageMagick/discussions/7375
https://github.com/ImageMagick/ImageMagick/discussions/7382
https://github.com/ImageMagick/ImageMagick/discussions/7430

**Reply by exekutive** [reply]
https://play.google.com/store/apps/details?id=com.paul.icon&hl=en_CA

**Comment by fmw42** [comment]
Have you seen https://github.com/ImageMagick/ImageMagick/discussions/6976
